app/src/main/java/com/flatstack/android/profile/ProfileViewModel.kt

```diff
@@ -18,11 +18,7 @@ class ProfileViewModel(
         loginRepository.onDestroy()
     }
 
-    fun updateProfile() {
-        profileBoundResource.fetchFromNetwork()
-    }
+    fun updateProfile() = profileBoundResource.fetchFromNetwork()
```

Comment:
Let's read on best practices of how to use specify return value in such blocks. I mean it's not clear does this function return something, or is it just function call.